dashboard/pages/primary_screening/callbacks.py

```diff
@@ -49,6 +56,129 @@ def upload_bmg_data(contents, names, last_modified, stored_uuid, file_storage):
     )
 
 
+# === STAGE 2 ===
+
+DISPLAYED_PLATES = 9
```

Comment:
visual suggestion: I would show the plates on 3x4 grid and decrease the size of the barcode font to reduce gaps

Reply:
Its a good Idea but unfortunatly it significantly impacts the styling (figure and table do not want to overflow the page size and it forces page to be scrollable):

![image](https://github.com/zuzg/drug-screening/assets/72276326/ddd758ae-b786-4acb-a3a8-58817644ac11)

Reply:
+ the table will have to be scrollable to fit

Reply:
In my opinion no scrollable page and table look much better and allows for easier analysis

Reply:
Okay, I see. We can leave it for now then and wait for client's feedback